devtools/credits.pl

```diff
@@ -1039,7 +1039,7 @@ sub add_paragraph {
 
 			begin_section("GUI");
 				add_person("Max Horn", "Fingolfin", "(retired)");
-				add_person("Vicent Marti", "tanoku", "");
+				add_person("Vicent Marti", "tanoku", "(retired)");
```

Comment:
One could of course also point out that it was (initially) a GSoc task, but I don't remember in which year etc., so I'll leave that to somebody else, if it is desired.

Reply:
Yes, that's a good point. He was never part of the team full time. And we do not mark one time contributors, which we have a few, as retired.

Also, could you please check your e-mail, I sent you a request regarding Google Analytics.